modules/rainstorm/1.0/config/default.yaml

```diff
@@ -1,30 +1,37 @@
 lcr-modules:
-    
+
     rainstorm:
 
         inputs:
-            # Available wildcards: {cohort_name}
-            sample_maf: "__UPDATE__" # path to the maf file containing all patients across cohort of interest
+            # Available wildcards {genome_build} from options below.
+            master_maf: "__UPDATE__", # This module only runs on genome data
+            subsetting_categories: "__UPDATE__" # e.g. "data/metadata/level3_subsetting_categories.tsv"
+        sample_set: ["__UPDATE__"] # E.g. ["DLBCL-only", "BL_DLBCL-BL-like_all"]
+        # Uncomment the following line to add a launch_date. Otherwise the current year-month will be used
+        # launch_date: ""
 
         scratch_subdirectories: []
 
+        include_non_coding: True
+        prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.1/generate_smg_inputs.R"
+
         options:
-            cohort_name: "__UPDATE__" # this will be used as prefix to the output files
-            rainstorm_version: 0.3 # here the version tag for the rainstorm version can be specified. The default version used in the module is 0.3
-            genome_build: "__UPDATE__" # specify genome build of variants in maf file
+            rainstorm_version: 0.4 # here the version tag for the rainstorm version can be specified.
+            genome_build: "__UPDATE__" # specify genome build of variants in maf file, hg38 or grch37
```

Comment:
Maybe the example from the PR can also be added here as comment?
`["hg38", "grch37"]`
so that the user knows it can be  a list